src/Device.cpp

```
@@ -0,0 +1,32 @@
+std::vector<bool> Device::getDeviceStatus() const {
+	return deviceStatus;
+}
+
+void Device::updateDeviceStatus(int deviceIndex, bool status) {
+	if (deviceIndex >= 0 && deviceIndex < deviceStatus.size()) {
+		deviceStatus[deviceIndex] = status;
+		isDeviceChanged = true;
+	}
+}
+
+void Device::sendDeviceStatus() {
```

Cppcheck (2.10-2)

[style] The function 'sendDeviceStatus' is never used.

(unusedFunction)